storage/ndb/src/kernel/blocks/dbtc/DbtcInit.cpp

```diff
@@ -1,6 +1,6 @@
 /*
    Copyright (c) 2003, 2023, Oracle and/or its affiliates.
-   Copyright (c) 2021, 2023, Hopsworks and/or its affiliates.
+   Copyright (c) 2021, 2025, Hopsworks and/or its affiliates.
```

Comment:
You changed copyright but no content. Did you mean to initialize `c_use_query_thread_for_locked_reads`?